app/schemas/com.example.stromtracker.database.AppDatabase/1.json

```diff
@@ -75,8 +75,67 @@
           ],
           "autoGenerate": true
         },
-        "indices": [],
-        "foreignKeys": []
+        "indices": [
+          {
+            "name": "index_Geraete_kategorieID",
+            "unique": false,
+            "columnNames": [
+              "kategorieID"
+            ],
+            "createSql": "CREATE INDEX IF NOT EXISTS `index_Geraete_kategorieID` ON `${TABLE_NAME}` (`kategorieID`)"
+          },
+          {
+            "name": "index_Geraete_haushaltID",
+            "unique": false,
+            "columnNames": [
+              "haushaltID"
+            ],
+            "createSql": "CREATE INDEX IF NOT EXISTS `index_Geraete_haushaltID` ON `${TABLE_NAME}` (`haushaltID`)"
```

Comment:
Heißt dass, das du den Index in der Geräte tabelle erstellst oder wird die trotzdem im haushalt erstellt. Da man ja erst ein Haushalt braucht, um ein Gerrät hinzuzufügen.

Reply:
Der JSON Code wurde generiert, dazu speziell kann ich dir nichts sagen.
Aber ich erstelle Indizes in der Geräte Entität auf die Fremdschlüssel der Geräte Tabelle. Muss nicht sein, aber wird einem ziemlich deutlich vorgeschlagen, dass das deutlich effizienter ist.